src/main/kotlin/it/pagopa/ecommerce/helpdesk/dataproviders/oracle/PmQueries.kt

```diff
@@ -151,3 +151,83 @@ val userFiscalCodeCountQuery =
         AND pu.STATUS IN ('11', '12')
     """
         .trimIndent()
+
+val searchWalletByUserFiscalCode =
+    """
+    SELECT  pu.FISCAL_CODE , pu.NOTIFICATION_EMAIL , pu.SURNAME , pu.NAME , pu.USERNAME , 
+		CASE pu.STATUS 
+		WHEN 0 THEN 'Utente non registrato'
+		WHEN 1 THEN 'Utente registrato non SPID'
+		WHEN 2 THEN 'Utente in attesa di verifica OTP'
+		WHEN 3 THEN 'Password da impostare'
+		WHEN 4 THEN 'Password da impostare - cambio password'
+		WHEN 5 THEN 'Utente cancellato'
+		WHEN 11 THEN 'Utente registrato SPID'
+		WHEN 12 THEN 'Utente registrato su IO con CIE'
+		ELSE TO_CHAR(pu.STATUS) END AS STATUS_UTENTE,
+		pw."TYPE" ,
+		pw.CREATION_DATE , pp.ID_PSP , pp.BUSINESS_NAME ,
+		pw.FK_CREDIT_CARD, pcc.CARD_BIN , pcc.CARD_NUMBER ,
+		pw.FK_BUYER_BANK, pmbb.ALIAS as MYBANK, pmbb.STATE AS MYBANK_STATE ,
+		pw.FK_BANCOMAT_CARD, pbc.ABI as BANCOMAT_ABI, pbc.CARD_PARTIAL_NUMBER AS BANCOMAT_NUMBER,
+		ps.ID_SATISPAY AS FK_SATISPAY,
+		pw.FK_BPAY, pb.BANK_NAME AS BPAY_NAME ,pb.CELLPHONE_NUMBER AS BPAY_NUMBER,
+		pw.FK_GENERIC_INSTRUMENT, pgi.DESCRIPTION AS GENERIC_INSTRUMENT_DESCRIPTION,
+		pp2.ID  AS FK_PPAL, pp2.EMAIL_PP AS PPAL_EMAIL 
+        FROM AGID_USER.PP_USER pu 
+        left JOIN AGID_USER.PP_WALLET pw ON pu.ID_USER =pw.ID_USER 
+        left  JOIN AGID_USER.PP_CREDIT_CARD pcc  ON pw.FK_CREDIT_CARD  = pcc.ID_CREDIT_CARD 
+        left JOIN AGID_USER.PP_MYBANK_BUYER_BANK pmbb  ON pw.FK_BUYER_BANK  = pmbb.ID_BUYER_BANK 
+        left JOIN AGID_USER.PP_BANCOMAT_CARD pbc  ON pw.FK_BANCOMAT_CARD  = pbc.ID_BANCOMAT_CARD 
+        left JOIN AGID_USER.PP_SATISPAY ps  ON pw.FK_SATISPAY = ps.ID_SATISPAY AND pw.FK_SATISPAY IS NOT null --pw.ID_USER  = ps.ID_USER and  pw."TYPE"  ='3'
```

Comment:
```suggestion
        left JOIN AGID_USER.PP_SATISPAY ps  ON pw.FK_SATISPAY = ps.ID_SATISPAY AND pw.FK_SATISPAY IS NOT null
```
remove this query comment

Reply:
Fixed with 23580a36a8e5d732bc23936bbd4a78d39a7e3cfb